definition-map.js

```diff
@@ -40,7 +40,11 @@ const QUERIES = {
     CourseById: {
         metadescription: "Returns various information about the course and the objects that are contained inside of the course, such as the sections, lessons, and pages associated with the course.",
         args: {
+<<<<<<< Updated upstream
```

Comment:
Looks like some git markers were committed in the code, will need to remove these.